README.md

```diff
@@ -15,7 +15,10 @@ This is the open-source portion of the back-end, website scraping software that
         PROPRIETARY_SITE_SCRAPERS_PATH="./../proprietary/site-scrapers" (optional, example)
 ## Using this code
 1. In your terminal, install dependencies with `npm install`
-1. To run: `node main.js`
+1. To run all scrapers: `node main.js`
+To run an individual scraper, specify the base filename from site-scrapers, e.g.:
+`node main.js MAImmuizations`
```

Comment:
missing an N

Reply:
Oh, the absolute pain of haste. And then I got distracted 4 hours ago. Fixed, sorry.
Plus a bonus.